Felienne/hedy
# [FEATURE] Improves landing page into general starting point on login

**Description**
In this PR we make some small changes to the landing page to re-introduce this page as a landing page on each login instead of only on account verification. The content is extended with the general user statistics as well as the last saved program. We also generalize the text for starting the tutorial or simply starting programming.

**Fixes**
This PR fixes #2773.

**How to test**
When logged in navigate to `/landing-page` and verify that the page contains some general user statistics and information. When visiting for the first time only the three select options are shown.
